doc/examples/segmentation/plot_random_walker_segmentation.py

```diff
@@ -44,7 +49,7 @@
 ax1.axis('off')
 ax1.set_adjustable('box-forced')
 ax1.set_title('Noisy data')
-ax2.imshow(markers, cmap='hot', interpolation='nearest')
+ax2.imshow(markers, cmap='viridis', interpolation='nearest')
```

Comment:
How about `magma` here? Although viridis is a good colormap, I think for images (with nothing superimposed and no foreground/background), `magma` is better because it spans the full brightness range (black to white), instead of going from purple to yellow.